hugo/content/tutorials/langium_and_monaco.md

```diff
@@ -169,233 +214,368 @@ h1 {
     margin: 8px auto;
     text-align: center;
 }
+#page-wrapper {
+    display: flex;
+    max-width: 2000px;
+    margin: 4px auto;
+    padding: 4px;
+    min-height: 75vh;
+    justify-content: center;
+}
+#page-wrapper .half {
+    display: flex;
+    width: 40vw;
+}
+.build {
+    display: block;
+    margin: 8px auto;
+    width: 300px;
+    height: 30px;
+    background: none;
+    border: 2px #fff solid;
+    color: #fff;
+    transition: 0.3s;
+    font-size: 1.2rem;
+    border-radius: 4px;
+}
+.build:hover {
+    border-color: #6cf;
+    color: #6cf;
+    cursor: pointer;
+}
+.build:active {
+    color: #fff;
+    border-color: #fff;
+}
 footer {
     text-align: center;
     color: #444;
     font-size: 1.2rem;
     margin-bottom: 16px;
 }
 @media(max-width: 1000px) {
+    #page-wrapper {
+        display: block;
+    }
+    #page-wrapper .half {
+        display: block;
+        width: auto;
+    }
     #minilogo-canvas {
         margin-top: 32px;
     }
+    #page-wrapper {
+        min-height: auto;
+    }
 }
+
 /* for monaco */
 .wrapper {
     display: flex;
     flex-direction: column;
     height: 100%;
     width: 100%;
 }
+
 #monaco-editor-root {
     flex-grow: 1;
 }
+
+#status-msg {
+    color: red;
+}
 ```
 
-Finally, there's the actual Javascript setting up our Monaco instance (stored in **src/static/setup.js**), and for setting up Langium as well. This is the most complex part of setting up Langium + Monaco in the web, so we'll walk through the file in parts.
+Finally, there's the actual Javascript setting up our Monaco instance (stored in **src/static/minilogo.ts**), and for setting up Langium as well. This is the most complex part of setting up Langium + Monaco in the web, so we'll walk through the file in parts.
+
+(*Update on Oct. 4th, 2023: Previously we wrote this as **src/static/setup.js**. This new file can be considered the same, but reworked into TypeScript & updated for the new versions of Langium & the MER.*)
 
 First, we need to import and setup the worker, as well as some language client wrapper configuration.
 
-```js
-import { MonacoEditorLanguageClientWrapper } from './monaco-editor-wrapper/index.js';
-import { buildWorkerDefinition } from "./monaco-editor-workers/index.js";
+```ts
+import { MonacoEditorLanguageClientWrapper, UserConfig } from "monaco-editor-wrapper/bundle";
+import { buildWorkerDefinition } from "monaco-editor-workers";
+import { addMonacoStyles } from 'monaco-editor-wrapper/styles';
 
-buildWorkerDefinition('./monaco-editor-workers/workers', new URL('', window.location.href).href, false);
+/**
+ * Prepare to setup the wrapper, building the worker def & setting up styles
```

Comment:
This comment should be updated. This is basically setting up monaco's own workers and necessary style for monaco-editor.